### PR TITLE
[Translation] [russian] - generics-struct-types

### PR DESCRIPTION
**Pull request by alexshd**

### CODE REVIEW

File: _content/tour/rus/generics-struct-types.article [hunk 1]
```diff
@@ -112,32 +110,166 @@ Output:
...
+.play generics/struct-types/answer1.goСтруктурные типы
+Вы можете объявить дженерик, используя  struct.
+
+* Generics - Struct Types
+
+Вы можете объявить дженерик, используя  `struct`.
+
+** Видео
+
+Посмотрите лекцию о генериках, в которой я рассказываю обо всех
```
Comment (shalimski) [inline _content/tour/rus/generics-struct-types.article:147]:
```suggestion
Посмотрите лекцию о дженериках, в которой я рассказываю обо всех
```

File: _content/tour/rus/generics-struct-types.article [hunk 2]
```diff
@@ -1,57 +1,55 @@
...
 
-You can declare a generic type using a struct type.
+Вы можете объявить дженерик, используя  `struct`.
 
-** Video
+** Видео
 
-Watch the talk I gave about Generics which walks you through all the
-examples in this section of the Tour.
+Посмотрите лекцию о генериках, в которой я рассказываю обо всех
```
Comment (shalimski) [inline _content/tour/rus/generics-struct-types.article:10]:
```suggestion
Посмотрите лекцию о дженериках, в которой я рассказываю обо всех
```

File: _content/tour/rus/generics-struct-types.article [hunk 3]
```diff
@@ -1,57 +1,55 @@
...
+примерах в этом разделе.
 
 .html generics-video.html
 
-** Code Review
+** Обзор кода
 
-- *Example* *1*: Linked List
+- *Пример* *1*: Связанный список
```
Comment (shalimski) [inline _content/tour/rus/generics-struct-types.article:17]:
```suggestion
- *Пример* *1*: Связный список
```

File: _content/tour/rus/generics-struct-types.article [hunk 4]
```diff
@@ -112,32 +110,166 @@ Output:
...
+** Видео
+
+Посмотрите лекцию о генериках, в которой я рассказываю обо всех
+примерах в этом разделе.
+
+.html generics-video.html
+
+** Обзор кода
+
+- *Пример* *1*: Связанный список
```
Comment (shalimski) [inline _content/tour/rus/generics-struct-types.article:154]:
```suggestion
- *Пример* *1*: Связный список
```

**Review by shalimski — APPROVED** [review]
lgfm

**Comment by ardan-bkennedy** [comment]
@shalimski any chance you can run the russian branch and check over all the translations?

**Comment by shalimski** [comment]
> @shalimski any chance you can run the russian branch and check over all the translations?

Ok, i'll do it.

**Comment by ardan-bkennedy** [comment]
@shalimski once you accept the invite, I will give you access to the Russian branch and you can approve your own PRs

**Review by rumyantseva — APPROVED** [review]
@shalimski thank you so much for taking over the rest of the work! Unfortunately, I'm not super productive here at the moment. I like the idea of finishing and merging everything here :)  

I think, with your fresh look, we should have a good enough translation here.

**Pull request merged**